packages/office-ui-fabric-react/src/components/Persona/Persona.scss

```diff
@@ -455,7 +455,7 @@ $ms-Persona-presenceSizeXl: 28px;
 
   .imageArea,
   .image {
-    flex: 0 0 $ms-Persona-size28;
+    flex: 0 0 auto;
```

Comment:
It seems like we're duplicating this over and over. can we just put it in one place?

Reply:
I think so. Since we have width defined, we can just change to auto, put it in a general place and make it respect width. Thank you for advice.